dockerfile/sdk/native.Dockerfile

```diff
@@ -36,28 +31,27 @@ RUN [ ! -z "$PRE_BUILD" ] && sh -c "${PRE_BUILD}"; \
     [ ! -z "$BUILD_ENV" ] && export ${BUILD_ENV}; \
     [ ! -z "$BUILD_TAGS" ] && export "${BUILD_TAGS}"; \
     [ ! -z "$BUILD_DIR" ] && cd "${BUILD_DIR}"; \
-    make ${BUILD_TARGET}
+    LDFLAGS='-linkmode external -extldflags "-static"' make ${BUILD_TARGET}
 
 RUN mkdir /root/bin
 ARG BINARIES
 ENV BINARIES_ENV ${BINARIES}
 RUN bash -c 'BINARIES_ARR=($BINARIES_ENV); for BINARY in "${BINARIES_ARR[@]}"; do cp $BINARY /root/bin/ ; done'
 
-FROM alpine:edge
-
-LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/heighliner"
+RUN addgroup --gid 1025 -S heighliner && adduser --uid 1025 -S heighliner -G heighliner
 
-RUN apk add --no-cache ca-certificates jq curl git gcc nano lz4 wget unzip
```

Comment:
Removing all container utils

Reply:
will we still have basic sh/ash utils like `sleep` ?

Reply:
no, that would have to happen in an initContainer

Reply:
Update: since we decided to bring in the minimal shell and utils, the following are included: `sh`, `pwd`, `ls`, `cat`, `less`, `grep`, and `sleep`